Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -47,6 +47,15 @@ public InMemorySQLite()
         }
     }
 
+    /// <summary>
+    /// Result of deleting (or attempting to delete) an Order in a FakeDatabase
+    /// </summary>
+    public enum FakeDatabaseDeleteOrderResult
+    {
+        OrderWasAlreadyDeleted,
+        OrderSuccessfullyDeleted,
+        OrderWasNotFound
+    }
```

Comment:
FakeBookingSystem uses a different enum from OrderStore because (1). This project (OpenActive.FakeDatabase.NET) doesn't reference OpenActive.Server.NET and (2). Within this project, there are more possible return scenarios than may be available to another possible OrderStore implementation. It's possible to have an OrderStore implementation that would have no need of `OrderWasAlreadyDeleted`